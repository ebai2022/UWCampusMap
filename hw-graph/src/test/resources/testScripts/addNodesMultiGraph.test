# Test adding nodes to multiple graphs

# Create multiple graphs
CreateGraph A
CreateGraph B

# Add nodes
AddNode B e1
AddNode A e2
AddNode A e3
AddNode B e4

# Check if the nodes have been added
ListNodes A
ListNodes B