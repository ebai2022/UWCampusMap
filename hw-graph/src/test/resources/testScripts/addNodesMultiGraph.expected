# Test adding nodes to multiple graphs

# Create multiple graphs
created graph A
created graph B

# Add nodes
added node e1 to B
added node e2 to A
added node e3 to A
added node e4 to B

# Check if the nodes have been added
A contains: e2 e3
B contains: e1 e4